dev_guide/managing_images.adoc

```diff
@@ -354,7 +360,12 @@ installed on your system:
 +
 ----
 $ docker login -u <user_name> -e <email_address> \
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
```

Comment:
Atomic Registry uses this doc too, so we should add `atomic-registry` to the list.

http://docs.projectatomic.io/registry/latest/dev_guide/managing_images.html